tmt/steps/provision/podman.py

```diff
@@ -118,6 +118,32 @@ def pull_image(self) -> None:
             message=f"Pull image '{self.image}'."
             )
 
+    def _setup_network(self) -> list[str]:
+        """
+        Set up the desired network.
+        Will look for existing network using the tmt workdir name,
+        or will create that network if it doesn't exist.
+        Returns the network arguments to be used in podman run command.
+        """
+
+        run_id = self._tmt_name().split('-')[1]
+        self.network = f"tmt-{run_id}-network"
+
+        try:
+            self.podman(
+                Command('network', 'create', self.network),
+                message=f"Creating network '{self.network}'."
+                )
+        except tmt.utils.RunError as err:
+            if err.stderr and 'network already exists' in err.stderr:
+                # error string:
+                # https://github.com/containers/common/blob/main/libnetwork/types/define.go#L19
+                pass
```

Comment:
Thanks for fixing the issue, but, one more nitpicky comment: a warning would be nice :) I don't know if it's necessary, but I for one would add it, to let the user know there was a small pebble we decided to ignore.

Reply:
The thing is that the warnings would appear in all multihost jobs because

 * whenever the `provision` step is called for the second phase `network already exists`
 * and when `finish` is called for the first phase the `network is being used`

So... what about a debug log instead which could be used when investigating?

Reply:
Hm, true. `debug` it is then.

Reply:
Ok, done in a74cd555.